paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/bacs/BacsMandateButton.kt

```diff
@@ -0,0 +1,63 @@
+package com.stripe.android.paymentsheet.paymentdatacollection.bacs
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.material.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.unit.dp
+import com.stripe.android.uicore.stripeColors
+
+@Composable
+internal fun BacsMandateButton(type: BacsMandateButtonType, label: String, onClick: () -> Unit) {
+    val colors = when (type) {
+        BacsMandateButtonType.Primary -> ButtonDefaults.buttonColors(
+            backgroundColor = MaterialTheme.colors.primary,
+            contentColor = MaterialTheme.colors.onPrimary
+        )
+        BacsMandateButtonType.Secondary -> ButtonDefaults.buttonColors(
+            backgroundColor = MaterialTheme.stripeColors.component,
+            contentColor = MaterialTheme.stripeColors.onComponent
+        )
+    }
+
+    return TextButton(
+        modifier = Modifier
+            .fillMaxWidth()
+            .height(44.dp)
```

Comment:
Does this still work when the user has changed the font size on their phone?

Reply:
It does but definitely doesn't look great. This behavior for `height` is equivalent to what `PrimaryButton` does so they end up with similar results. Should we look into changing this with padding instead and have a minimum height of `44.dp` for both this button & `PrimaryButton`?

<img src="https://github.com/stripe/stripe-android/assets/141707240/0302dd70-a5ae-4e4f-9386-2dd7eb458315" width="40%" height="40%" />

Reply:
I think this is fine for now. Definitely worth thinking about, but not for this PR.